app/models/service/containers.rb

```diff
@@ -24,6 +24,10 @@ def self.start_container!(wizard_state)
       end
     end
 
+    def self.pull_image(container)
+      container.compute_resource.create_image(:fromImage => container.repository_pull_url)
```

Comment:
This fixes image retrieval for standalone registries, but not going to work in general case for non-standalone registries (basically normal operation when working with the hub), see https://docs.docker.com/reference/api/hub_registry_spec/#pull. Not only we need to query the index for the location of the image, we also must use the authentication token that the index passes to us together with the response.